lib/Pool.js

```diff
@@ -187,9 +189,7 @@ Pool.prototype._next = function () {
       if (task.resolver.promise.pending) {
         // send the request to the worker
         var promise = worker.exec(task.method, task.params, task.resolver)
-          .then(function () {
-            me._next(); // trigger next task in the queue
-          })
+          .then(me._boundNext)
```

Comment:
I noticed this callback was causing a decent amount of extra GC, so now we always reuse the same callback instance.